src/main/java/org/springframework/up/UpCliApplication.java

```diff
@@ -17,13 +17,17 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.up.config.TemplateRepositoryProperties;
+import org.springframework.up.config.UpCliProperties;
 
 /**
  * Main boot app.
  *
  * @author Janne Valkealahti
  */
 @SpringBootApplication
+@EnableConfigurationProperties({ UpCliProperties.class, TemplateRepositoryProperties.class })
```

Comment:
Keep main class tidy. Should move this to new `UpCliConfiguration` class like it was done in my PR.